### PR TITLE
In 1831

### PR DESCRIPTION
**Pull request by jdearie**

### CODE REVIEW

**Comment by jdearie** [comment]
In addition to creating the view block display and setting block viz, I created the pathauto pattern. While working on view, kept hitting error using exposed filters block with layout settings. found patch and applied - fixed issue.

**Comment by jdearie** [comment]
Manually deleted a bunch of the QAPP/SOP config that was accidentally merged locally.

File: config/sync/core.entity_form_display.node.hr_advisories_and_bulletins.default.yml
Comment (jmarruff) [inline config/sync/core.entity_form_display.node.hr_advisories_and_bulletins.default.yml:1]: Wasn't sure if you wanted policy identifier before the policy type option select. Not a biggie, just noted.

**Review by jmarruff — APPROVED** [review]
Looks fine to me.

**Pull request merged**